.travis.yml

```diff
@@ -8,6 +8,5 @@ install:
   - "pip install -r requirements.txt --use-wheel"
   - "pip install -r requirements-dev.txt --use-wheel"
 script:
-  - python manage.py syncdb --noinput
-  - python manage.py migrate
-  - python manage.py test
+  - flake8 .
```

Comment:
@imsickofmaps please, for the love of everything turning red in our editors, automate the pep8 checks in your $EDITOR.